gcp_variant_transforms/libs/vcf_header_parser.py

```diff
@@ -32,68 +32,29 @@
 HeaderFields = namedtuple('HeaderFields', ['infos', 'formats'])
 
 
-def get_merged_vcf_headers(input_pattern):
-  """Returns merged VCF headers (FORMAT and INFO) from ``input_pattern``.
-
-  If multiple files are specified by ``input_pattern`` then header fields will
-  be merged by key and only one will be chosen as representative (in no
-  particular order). If the same key is used in multiple files, then all types
-  and numbers must be the same.
+def get_vcf_headers(input_file):
+  """Returns VCF headers (FORMAT and INFO) from ``input_file``.
 
   Args:
-    input_pattern (str): A string specifying the path to VCF input file(s). They
-      can be local or remote (e.g. on GCS).
+    input_file (str): A string specifying the path to a representative VCF
+    file. It can be local or remote (e.g. on GCS).
```

Comment:
I think it would be good to clarify why the VCF file is representative here. i.e. the VCF file contains a header representative of all VCF files matching the input_pattern of the job.

Reply:
Done!